converter/shieldhit/geo.py

```diff
@@ -2,6 +2,7 @@
 from abc import ABC
 from dataclasses import dataclass, field
 from math import log10, ceil, isclose
+from scipy.spatial.transform import Rotation as R
```

Comment:
we are planning to include `yaptide/converter` in frontend using pyodide 
adding full scipy just to have rotation library is a bit of overkill.
I'd recommend to find somewhere the rotation code in python with good licence and paste it inside our codebase, then remove scipy from requirements.

Reply:
Could we leave it for now and change this when we integrate `pyodide`?

Reply:
I'd better do it now, postponing it may end up it repeating all tests and refactoring. It shouldn't be that hard to find such code.

Reply:
For example:
https://github.com/alisterburt/eulerangles
https://github.com/NOhs/mgen/blob/master/mgen/rotation_matrix_3d.py

I'd anyway strip those code from numpy deps

Reply:
Apparently we can use SciPy in pyodide.
Source: https://pyodide.org/en/stable/

Reply:
Calm down @grzanka 
Imo it srsly can be done later and what is more important in NEW PR

Reply:
Pyodide can use SciPy.
Source: https://pyodide.org/en/stable/

Reply:
Personally I really need fixes on this PR for https://github.com/yaptide/yaptide to properly run `shieldhit` instead of `dummy` simulation

Reply:
@p1003 you haven't mentioned before you need it for `dummy` simulation. If that it the case, then add an issue and lets do it later.

Reply:
@grzanka I need it for `shieldhit` to run, `dummy` is not working either but I am able to change it easly for testing

Reply:
'dummy' version has nearly empty `geo.dat` and I think that's why all results had `0.0` value